netci.groovy

```diff
@@ -206,13 +209,22 @@ def static getJobName(def configuration, def architecture, def os, def scenario,
 // Adds a trigger for the PR build if one is needed.  If isFlowJob is true, then this is the
 // flow job that rolls up the build and test for non-windows OS's.  // If the job is a windows build only job,
 // it's just used for internal builds
-def static addTriggers(def job, def branch, def isPR, def architecture, def os, def configuration, def scenario, def isFlowJob, def isWindowsBuildOnlyJob) {
+def static addTriggers(def job, def branch, def isPR, def architecture, def os, def configuration, def scenario, def isFlowJob, def isWindowsBuildOnlyJob, def doCrossgen) {
     if (isWindowsBuildOnlyJob) {
         return
     }
     
     // Non pull request builds.
     if (!isPR) {
+        //Pri 0 Crossgen tests run per-commit for all platforms in crosslist, plus Windows
+        if (doCrossgen) {
+            assert (os == 'Windows_NT' || os in Constants.crosslist)
+            //Add push triggers for all Checked x64 pri0 crossgen builds
+            if (configuration == 'Checked' && architecture == 'x64' && scenario == 'default') {
```

Comment:
For X64, we should do this for Checked and Release. I noticed that we have Windows x86 CI jobs as well. Can we please enable for them too?